quic/client/handshake/FizzClientHandshakeContext.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+#pragma once
+
+#include <quic/client/handshake/ClientHandshakeFactory.h>
+
+namespace quic {
+
+class FizzClientHandshake;
+
+class FizzClientHandshakeContext
```

Comment:
really sorry to be such a Pain about the naming stuff, we already have so many things named context, I just wanted to make sure we get it right.

How about FizzClientQuicHandshakeContext. I'd be cool with that :)

to make it clear that this is a quic specific context.

Reply:
ok